runtime/Game/Game.cpp

```diff
@@ -636,8 +636,8 @@ namespace spartan
                             renderable->SetInstances(instances);
 
                             // make the bark collidable
-                            shared_ptr<PhysicsBody> physics_body = bark->AddComponent<PhysicsBody>();
-                            physics_body->SetShapeType(PhysicsShape::Mesh);
+                            shared_ptr<PhysicsBody> physics_body_bark = bark->AddComponent<PhysicsBody>();
```

Comment:
Could this be that you are mistakenly redeclaring this here and you are actually hiding the outer one?
Could it be that this physics_body is supposed to be the same as in line 576 for example?

Reply:
They will still affect the same memory but the double declaration is a mistake, you can remove physics_body_bark if you want.